compass/ocean/tests/drying_slope/drying_slope.cfg

```diff
@@ -0,0 +1,27 @@
+# Options related to the vertical grid
+[vertical_grid]
+
+# Number of vertical levels
+vert_levels = 10
+
+# The type of vertical coordinate (e.g. z-level, z-star)
+coord_type = sigma
+
+# config options for drying slope test cases
+[drying_slope]
+
+# the number of grid cells in x and y
+nx = 6
+ny = 28
+
+# the size of grid cells (m)
+dc = 1000.0
```

Comment:
Similarly, this doesn't seem to be needed. Can `resolution` be passed as an argument to `self.add_step(InitialState(test_case=self))` so this can get set in `InitialState.run`?

Reply:
The same, if this is set in the code (in `configure()`), it shouldn't be set here, too.

Reply:
I'll defer to @xylar on whether it makes more sense to take the original approach of having these be config options that are modified in `Default.configure` or remove these as config options and set them in `InitialState.run` as you suggest. My thinking behind the use of config options was making these dimensions more externally visible, but I can see how it might be misleading since `dc` and `ny` are not independent.

Reply:
I think it's better to include it in configure() and not in the config file, to make a bit clearer that the configure() version always takes priority. You can use the brand new "comment" argument to set() to have the comment included there instead.

Reply:
Thanks @xylar! I didn't refresh in time so my comment was a bit lagged but it all worked out.